skimage/filters/_lic.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+
+
+def _astype_nocopy(a, dtype):
+    """
+    Workaround to enable "a.astype(dtype, copy=False) under Python 2.6
+    """
+    if a.dtype != dtype:
+        return a.astype(dtype)
+    else:
+        return a
+
+
+def line_integral_convolution(input, velocity, kernel, origin=0, order=3,
+                              weighted='average', step_size='unit_length',
+                              maximum_velocity=None):
+    """
+    Line integral convolution of an input image with an arbitrary kernel.
```

Comment:
One line short description, followed by a paragraph description.

The description at http://scipy-cookbook.readthedocs.io/items/LineIntegralConvolution.html is very nice; perhaps we can use that as inspiration.